CMakeLists.txt

```
@@ -139,6 +141,8 @@ target_link_libraries(engine-sim
 
 target_include_directories(engine-sim
     PUBLIC dependencies/submodules)
```

```suggestion
target_include_directories(engine-sim
    PUBLIC dependencies/submodules include)
```

You should add this to simplify include